lib/experimental/fonts/font-library/class-wp-rest-font-faces-controller.php

```diff
@@ -0,0 +1,461 @@
+<?php
+/**
+ * REST API: WP_REST_Font_Faces_Controller class
+ *
+ * @package WordPress
+ * @subpackage REST_API
+ * @since 6.5.0
+ */
+
+if ( class_exists( 'WP_REST_Font_Faces_Controller' ) ) {
+	return;
+}
+
+/**
+ * Class to access font faces through the REST API.
+ */
+class WP_REST_Font_Faces_Controller extends WP_REST_Posts_Controller {
+	/**
+	 * The base of the parent Font Family controller's route.
+	 *
+	 * @since 6.5.0
+	 * @var string
+	 */
+	private $parent_base;
+
+	/**
+	 * Parent font family post type
+	 *
+	 * @since 6.5.0
+	 * @var string
+	 */
+	private $parent_post_type;
+
+	public function __construct() {
+		$post_type       = 'wp_font_face';
+		$this->post_type = $post_type;
+
+		$post_type_obj   = get_post_type_object( $post_type );
+		$this->rest_base = $post_type_obj->rest_base;
+
+		$parent_post_type       = 'wp_font_family';
+		$this->parent_post_type = $parent_post_type;
+		$parent_post_type_obj   = get_post_type_object( $parent_post_type );
+		$this->parent_base      = $parent_post_type_obj->rest_base;
+		$this->namespace        = $parent_post_type_obj->rest_namespace;
+	}
+
+	/**
+	 * Registers the routes for posts.
+	 *
+	 * @since 6.5.0
+	 *
+	 * @see register_rest_route()
+	 */
+	public function register_routes() {
```

Comment:
This too should just use the parent classes register_routes method. None of this is needed.